Dockerfile

```diff
@@ -0,0 +1,14 @@
+ARG TARGET_VERSION=2.5.3
+
+FROM wpolicarpo/activerecord-sqlserver-adapter:${TARGET_VERSION}
```

Comment:
Probably not great to depend on a Docker image that you maintain separately. Can the whole Dockerfile be included here?

Reply:
As I said in the PR, I'd be happy to move to https://github.com/rails-sqlserver/docker-images if requested.